src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -302,6 +334,12 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
               Collapse direct recursion
             </MenuItem>
           : null}
+        {isCollapsed ? <div className="react-contextmenu-separator" /> : null}
+        {isCollapsed
+          ? <MenuItem onClick={this.handleClick} data={{ type: 'expand-all' }}>
+              Expand all
+            </MenuItem>
+          : null}
```

Comment:
I don't think we should hide it when it's collapsed -- some children might not be expanded and it could be easier to select any parent, right click, and expand from there.